.github/workflows/update_translations_from_source.yml

```diff
@@ -0,0 +1,34 @@
+name: Update Translations from Source
+
+on:
+  push:
```

Comment:
We should probably just do this as cron. Otherwise this will run when translations update and get pushed to master creating a possible loop.

Reply:
It only commits and pushes when there's a change, so it shouldn't do.